### ISSUE TITLE
Update code-server download to use new code-server v3 packaging

### ISSUE THREAD

**Issue by nathanpotter** [cdr/sshcode#167]
Code server is no longer packaged as a static binary and is instead packaged as a tarball. We should update sshcode to download from the new release URL:
`https://codesrv-ci.cdr.sh/releases/latest/linux-x86_64.tar.gz`

We can then untar that to `/usr/lib/code-server` and symlink the executable to the correct location.

**Comment by Merith-TK** [comment]
that would require sudo, which a user using sshcode may not have on the
server they are connecting to,
—Merith

**Comment by Merith-TK** [comment]
if anything, it should be extracted to $HOME/.cache/sshcode/
—Merith

**Comment by nathanpotter** [comment]
Yeah that sounds good as well, we should just untar it wherever is reasonable

**Comment by Merith-TK** [comment]
Would take too long, and affect user friendly ness, but would be easier in the long run

**Comment by gyzerok** [comment]
I seem to have a working solution here: https://github.com/gyzerok/sshcode/commit/9e0d09428836dcbd8c5bffd58461751cf7a4ab06. Can't open a PR yet, since it is dependent on another PR from me, which is currently open. However I welcome any feedback.

**Comment by israelpgjm** [comment]
> Code server is no longer packaged as a static binary and is instead packaged as a tarball. We should update sshcode to download from the new release URL:
> `https://codesrv-ci.cdr.sh/releases/latest/linux-x86_64.tar.gz`
> 
> We can then untar that to `/usr/lib/code-server` and symlink the executable to the correct location.

**Comment by Merith-TK** [comment]
> > Code server is no longer packaged as a static binary and is instead packaged as a tarball. We should update sshcode to download from the new release URL:
> > `https://codesrv-ci.cdr.sh/releases/latest/linux-x86_64.tar.gz`
> > We can then untar that to `/usr/lib/code-server` and symlink the executable to the correct location.

Please do not simply repost the original message
